### ISSUE TITLE
New tick interface doesn't provide option to log an RP ascent of an Alpine route

### ISSUE THREAD

**Issue by lordyavin** [theCrag/website#4249]
![grafik](https://github.com/theCrag/website/assets/1528540/a59aabf2-81e3-4030-b40f-217fb1525951)

![grafik](https://github.com/theCrag/website/assets/1528540/209b9e01-d161-44f5-ab02-b3f761925214)

I expect that an alpine route can be logged as redpointed

**Comment by braeuni** [comment]
A few points I'd like to address:

1. To my understanding, Eternal Flame isn't an "alpine" route in theCrag's meaning of the term "alpine". It's a (Mixed) Trad route.
2. The route type "alpine" on theCrag often leads to confusion in german speaking users. The term "Alpinklettern" more or less means a (mixed)-trad route in an alpine environment. Alpine routes on theCrag are "(kombinierte) Hochtouren".  
3. ~~Thus it's correct that an alpine route cannot be redpointed~~. You should change the type of the route. See also #4220

**Comment by lordyavin** [comment]
Please specify your term of "mixed-trad". I don't see why #4220 is related.

**Comment by braeuni** [comment]
https://www.thecrag.com/de/artikel/styles#gemischt-traditionelles-klettern
https://www.thecrag.com/de/artikel/styles#alpines-klettern-hochtouren

Aus dem gleichen Grund wurde auch https://www.thecrag.com/de/klettern/austria/route/701653350 geändert.

**Comment by lordyavin** [comment]
Okay I see, now I know why this issue triggered you. Are you [KiHa](https://www.thecrag.com/climber/kiha)? Never mind. You don't have to answer.

Back to my point of view. Why isn't it possible to redpoint an alpine route (kombinierte Hochtour)? The definition of theCrag is as follows: 
https://www.thecrag.com/en/article/ticktypes
> The following ascent types are applied in lead climbing which includes sport climbing, traditional climbing, ice climbing and deep water soloing.
> ...
> Redpoint: I led this route, without falling or resting, but not on my first attempt (incl. repeats).

 So I suppose that if there is a route that requires mixed equipment (crampons, ice axe, climbing shoes, mobile gear, bolts, ...) and I free climb every pitch, without falling or resting on lead, I should be able to claim a red-point ascent.

You could argue that climbing an ice passage with crampons & axe is aid climbing because we are using a tool to gain height. So what exactly do you mean when you say that it is not possible to red-point an alpine route?

In general, I think that any route that requires climbing on snow or ice (either on the approach, the route itself or the descent) is an alpine route. Because of the climate change it will be the case that more and more routes won't meet these requirements any more and we may have to change their gear style. But I wouldn't do that in advance.

**Comment by braeuni** [comment]
While I get you point, I think it's all a matter of the correct terminology.

> In general, I think that any route that requires climbing on snow or ice (either on the approach, the route itself or the descent) is an alpine route. Because of the climate change it will be the case that more and more routes won't meet these requirements any more and we may have to change their gear style. But I wouldn't do that in advance.

"Alpine route" is a pretty broad term. It's just that the term redpointing is not really used in this kind of setting. One does not redpoint Everest. 
"Alpine route" is an umbrella term where #4220 comes into play. On its sub levels, a grade like D+(S+) implies grade IV UIAA climbing and/or a certain degree of ice etc.
A deterministic representation would be a route like The Crystal Ship (1,600m, 6b A2 M7) where all individual climbing styles required are addressed and the individual pitches can be redpointed/onsighted/aided/etc.

The approach or descent has nothing to do with the route gear style. When you put up a 20 meter pitch with 10 bolts on Eiger, its also just a sport climb.
Again, the term "Alpinklettern" is misleading and not the same as an alpine route on theCrag. 
Just be cause a (rock) route is in an alpine environment doesn't make its gear style alpine, although you would call it "Alpinklettern"
That's why the term "Alpines Sportklettern"  (= gear style: sport) exists, to differentiate the level of protection.

P.S:
As I read this over and over, it seems to me, that it is not as clear as I'd like it to be.
Basically alpine route can mean everything and is a strong generalization.

**Comment by lordyavin** [comment]
> As I read this over and over, it seems to me, that it is not as clear as I'd like it to be.
> Basically alpine route can mean everything and is a strong generalization.

Acknowledged and appreciated

Please don't misunderstand, but could we leave the academic discussion behind and remain pragmatic? 

What use is the route information supposed to be? In my opinion, safety should come first. In other words, informing the climber about the requirements of the route in terms of ability and equipment. This also includes how to reach the start of the route and how to get back down from the "summit". Of course, you can now say that the ascent and descent and the necessary equipment are clear from the description of the area or are even implicit. But it is better for a route to be incorrectly labelled as alpine than for someone to go climbing in shorts, without weather protection and emergency equipment. Maybe I'm underestimating the average climber, but there are more and more of them and the more there are, the more idiots there will be. But I'm getting polemical...

> The approach or descent has nothing to do with the route gear style. When you put up 20 meter pitch with 10 bolts on eiger, its also just a sport climb.

It's not just an "ordinary" sport climb. It is a sport climb in a notorious north face in high altitude. It doesn't matter how long it is, there is much more danger and risk to attempt to climb this route as meeting your friends for a session in the Frankenjura. And that's why I prefer to mark such routes as alpine - but that is against the definition of theCrag.

Another point of view: What about the achievement of an athlete? It's difficult to say which is the greater achievement or which deserves more respect, but a "9+ alpine" has a completely different character to a "9+ sport". Right?

**Comment by lordyavin** [comment]
Okay okay, as you so I did:

>Alpine climbing is a style of climbing in which the primary aim is very often to reach the summit of a mountain typically through different terrain (rock, snow, ice). On theCrag we DO NOT consider climbs that are pure Trad , Mixed trad or Aid climbs Alpine climbs just because they are in an alpine setting.

Don't know when this was set, but there are exceptions to this definition. In high altitude the possibility for ice and snow exists (at least in the past). So we want to distinct between climbing (on vertical faces) and mountaineering (to reach a summit), don't we? Looks like, and I had the impression in the past, that the term alpine needs to be revised and potentially replaced with something that fits better.

**Comment by lordyavin** [comment]
@rouletout, @nicHoch and @scd what your thoughts on this?

**Comment by lordyavin** [comment]
Come on guys. Are you that busy?
